### PR TITLE
fix: permissions behavior

### PR DESCRIPTION
**Pull request by arzola**
This PR address #74

### CODE REVIEW

File: tests/Traits/CreatesModels.php [hunk 1]
```diff
@@ -114,4 +114,31 @@ public function createInstitutionsUsers(int $institutionsLimit, int $usersLimit)
...
+
+    /**
+     * This method runs a callback without triggering a given filter.
+     * It returns the expected value from the callback.
+     *
+     * @param  string  $hook The hook name to be skipped
+     * @param  callable  $callback The method that should run
+     * @return mixed The value from the callback method
+     */
+    protected function runWithoutFilter(string $hook, callable $callback): mixed
```
Comment (richard015ar) [inline tests/Traits/CreatesModels.php]: Considering CreatesModels trait was intended to create model instances for testing, I was wondering if this should be in a different trait/helper? or maybe for now just a method within the PermissionsManagerTest class until we see the need to re-use it on other tests, what do you think?

**Review by richard015ar — CHANGES_REQUESTED** [review]
I left a comment on the code. It is not mandatory, we can chat about it.

Although, as an IM I was not able to access to the site-info page of book as expected, I do have access to book dashboards that do not belong to my institution.

How I replicated it?
- I pick a book where the IM does not have any roles in it and that does not belong to any institution.
- I log in as that IM and pasted the book dashboard URL `<BOOK_URL>/wp-admin/index.php?page=book_dashboard` and I was able to access to it.
- I also can access to book dashboard that belong to other institutions, without any role in it.

**Comment by richard015ar** [comment]
I know it's outside the scope of the related issue, but are you willing to add to this PR the new name for this plugin in the main file `pressbooks-multi-institution.php`:
```php
/**
 * Plugin Name: Pressbooks Shared Network
...
 */
```
So it will be displayed with the new name on the Plugins list.

**Comment by arzola** [comment]
@richard015ar thank you for the feedback I couldn't replicate accessing to other non related books, also @tw77 tested that behaviour yesterday

**Review by richard015ar — CHANGES_REQUESTED** [review]
LGTM, it was a cache problem I think, now it works as expected. Sorry! Thank you for the changes requested

**Pull request merged**